requirements/snyk/libraries/requirements.txt

```diff
@@ -1 +0,0 @@
-../../libraries.txt
```

Comment:
This seems like a problematic side-effect of our symlinks, @Scotchester.

Reply:
Fascinating.